### DISCUSSION TITLE
Problem with my GitHub Sponsors Account

### DISCUSSION THREAD

**Discussion by BKDev99** [community/community#23967] [New to GitHub]
<p>I’m having a problem with my GitHub Sponsors account. My account was suddenly closed, and I have outstanding payments in Stripe account. The last time I saw billing timing information was March 22, and now March 24. I submitted a support request but have not received any response. How can I get this payment? Thank you.</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>We’re unable to deal with queries about account specific details in the public forums. I can confirm your ticket has been received and has been routed to the correct team.</p>

**Comment by BKDev99** [comment]
<p>Thanks for your response! I see the time it takes for a ticket to be resolved is 7-10 days. Is it correct?</p>

**Comment by canuckjacq** [comment]
<p>That’s probably a decent place to set an expectation. Support is quite a large organisation and we have a variety of teams and queues, so response times can vary substantially.</p>

**Comment by BKDev99** [comment]
<p>Sorry, I don’t want to be a spammer, but I don’t understand the process; in total, I sent 2 tickets, but I didn’t receive any reply emails, while my account balance changed continuity.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>If you’re not receiving your confirmation emails, then you probably need to go check your spam folders/any filters you have set up.</p>
<p>If you’ve received your confirmation but no reply, please wait. Some of these cases are complex and do take some time.</p>
